spec/requests/api/journalist_can_create_article_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::ArticlesController, type: :request do
+  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+
+  describe 'POST /api/articles_controller' do
+    before do
+      post '/api/articles/create',
+           params: {
+             title: 'Coronavirus',
+             teaser: 'Things are bad',
+             content: 'It will get worse.'
```

Comment:
IMO, the conventional way to send this off is wrapped in an `'article'` key. It will be parsed by your controller in that way and require you to sanitize params differently than you do at the moment.